src/binutil/mod.rs

```diff
@@ -7,7 +7,7 @@ pub mod signal_handler;
 pub fn tikv_version_info() -> String {
     let fallback = "Unknown (env var does not exist when building)";
     format!(
-        "\nRelease Version:   {}\
+        "Release Version:   {}\
```

Comment:
/cc @overvenus

Reply:
ping @overvenus

Reply:
Could you keep it? It is needed in `--version`

```
$ tikv-ctl --version
TiKV Control (tikv-ctl)
Release Version:   3.0.0-beta.1
$ tikv-ctl --version
TiKV Control (tikv-ctl) Release Version:   3.0.0-beta.1
```